Source/RapyutaSimulationPlugins/Public/Core/RRGameSingleton.h

```diff
@@ -26,6 +27,28 @@
 
 #include "RRGameSingleton.generated.h"
 
+template<const ERRResourceDataType InDataType>
```

Comment:
is this a standard way to define typenamne?

Reply:
I suppose so, `using` as introduced since c++11 is favored over `typedef`
https://www.geeksforgeeks.org/cpp-using-vstypedef
https://www.nextptr.com/tutorial/ta1193988140/how-cplusplus-using-or-aliasdeclaration-is-better-than-typedef